tigerpath/majors_and_certificates/majors/SOC_2018.json

```
@@ -6,7 +6,7 @@
   "year": 2018,
   "urls": [
     "https://ua.princeton.edu/academic-units/department-sociology",
-    "https://sociology.princeton.edu/"
+    "https://sociology.princeton.edu/undergraduate-program/current-majors/requirements"

```

Undergraduate handbook
```suggestion
    "https://sociology.princeton.edu/undergraduate-program/current-majors/requirements",
    "https://sociology.princeton.edu/sites/sociology/files/undergraduate_handbook_201920.pdf"
```